scripts/join_scenarios.py

```diff
@@ -41,7 +41,7 @@ def load_scalars(path):
         scalars = load_scalars(
             os.path.join("results", scenario, "postprocessed", "scalars.csv")
         )
-        scalars["scenario"] = scenario
+        scalars["scenario_key"] = scenario
```

Comment:
The postprocessed results are not labeled by scenario_key, but by scenario.

Reply:
Done in [0da658b](https://github.com/rl-institut/oemof-B3/pull/147/commits/0da658bd4567e29e1929d6f1cb95a9459975a051)